backend/src/main/java/com/karankumar/bookproject/backend/service/TagService.java

```diff
@@ -38,7 +38,19 @@ public Tag findById(Long id) {
     }
 
     public void save(@NonNull Tag tag) {
-        tagRepository.save(tag);
+        boolean hasDuplicate = false;
```

Comment:
Thanks for this! This works, but could you instead create a derived query that searches for a tag by name (that's case insensitive)? That would be useful in other places to find books that belong to a particular tag. 

I appreciate I didn't mention this before, but it just came to mind when looking at this method's implementation

Reply:
Fair enough, I added a findByName() method to TagRepository and this is referenced inside TagService. The test that I wrote for this seems to work as well.